keep-starknet-strange/beerus
# Feat: Wasm compatibility with no_std

# Pull Request type

Rebased beerus to be no_std / wasm compatible:

To test, run:

cargo build --no-default-features --target wasm32-unknown-unknown --release

Please check the type of change your PR introduces:


# What is the current behavior?



Issue Number: N/A

# What is the new behavior?



-
-
-

# Does this introduce a breaking change?




# Other information
